src/components/FullscreenModal/ScrollToTop/ScrollToTop.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+
+import { pxToRem } from '../../../utils/helpers';
+import { Button } from '../../Button';
+import { ButtonColors, ButtonVariants } from '../../Button/Button.enums';
+import { Icon } from '../../Icon';
+import { SSCIconNames } from '../../Icon/Icon.enums';
+import { ScrollToTopProps } from './ScrollToTop.types';
+
+const ScrollToTopButton = styled(Button)`
+  flex-direction: column;
+  justify-content: center;
+  height: ${pxToRem(48)};
+  padding: 0;
+  text-align: center;
+  line-height: ${pxToRem(13)};
+`;
+
+const ButtonText = styled.span`
+  margin-top: ${pxToRem(4)};
+`;
+
+const ScrollToTop: React.FC<ScrollToTopProps> = ({ onClick = undefined }) => {
```

Comment:
- nitpick but this should be named `ScrollToTopButton`
- unnecessary default value?